lib/wprof/reporters/external_report.rb

```diff
@@ -0,0 +1,13 @@
+module Wprof
+  module Reporters
+    module ExternalReport
+      def external_report
+        options = { body: @data }
+        headers = WProf::Config.get_value(:external_headers)
+        options.merge!(headers) unless headers.nil?
```

Comment:
```suggestion
        options.merge!(headers) if headers.present?
```

Reply:
I cannot accept this change because, .'present?' its a Rails method and I have a some trouble when I try testing. So, sorry, I'm agreed with that change but its a new rock in my way for tested.